subscription/upgrade.md

```diff
@@ -60,6 +60,6 @@ To update the billing information for your organization or personal account:
 
     > **Note**
     >
-    > The details displayed in the **Contact Information** section is used to generate your invoice. All billing-related information is sent to the email address listed in this section. Invoices are sent to one email address. Please contact your administrator to create a group within your organization that includes all of the email addresses.
+    > The details displayed in the **Contact Information** section is used to generate your invoice, and is the default information you provided when the subscription was created. All billing-related information is sent to the email address listed on the **Payment Methods** page in the **Contact Information** section. You can send emails to a maximum of two users in your organization. Contact your administrator to create a group within your organization that includes all of the email addresses.
```

Comment:
> You can send emails to a maximum of two users in your organization. Contact your administrator to create a group within your organization that includes all of the email addresses.

This doesn't make sense to me because an organization's members can't send emails, and it's not how it works.

To me, this note is overloaded because it mentions the billing address and email, which uses for different purposes. If we can move *Update billing information* into a separate page.

For examples:

```
Your personal account's billing email is where Docker Hub sends invoices and other billing-related communication. By default, your primary email address is used as the billing email address. However, you can change the billing email associated with your personal account at any time.

Your organization account's billing email is where Docker Hub sends invoices and other billing-related communication. By default, an organization creator's primary email address is used as the billing email address. However, you can change the billing email associated with your organization account at any time.

Additionally, organization members within the **owners** team are used as additional recipients for your organization's billing email.
```

Reply:
Thanks, @ob1dev Sure. I consolidated this note until the new topics are created for Billing. 
We discussed that you can have up to two emails for the Billing account.

Reply:
For the organization account, there could be are two groups of email addresses depending on specific email type:

1. billing - single email address that an organization owner can set up in the section `Contact Information`.
2. owners - multiple email addresses of all members of the group/team `owners` within an organization.

See the following link for more details:
https://docs.google.com/document/d/1GT4DyS9xX8q1sEkIX_AR3efI_ATRmNIVKHuclGKwbr8/edit